### PR TITLE
Replace @angular/flex-layout

### PR DESCRIPTION
**Pull request by coverbeck**
**Description**
Replaces `@angular/flex-layout` with `ngbracket/flex-layout`.

***Background***
Angular stopped updating `@angular/flex-layout` as of Angular 15. SEAB-6278 has several links with the background. This will cause us difficulties in upgrading to newer versions of Angular.

Using the module `ngbracket/flex-layout` is the path of least resistance. It is a fork of the `@angular/flex-layout` that is updated for newer versions of Angular. It seems to have decent traction in terms of npm downloads. The other main option is to go to a pure CSS implementation, using CSS Flexbox or tailwind, but those would be a bit more work. We should reconsider those for the future, but for now, to unblock us moving to an Angular 16 upgrade, I think this is best.

**Review Instructions**
Sanity check the main pages in the UI and make sure the layout looks unchanged.

**Issue**
SEAB-6278

**Security**
If there are any concerns that require extra attention from the security team, highlight them here.

Please make sure that you've checked the following before submitting your pull request. Thanks!

- [x] Check that your code compiles by running `npm run build`
- [x] Ensure that the PR targets the correct branch. Check the milestone or fix version of the ticket.
- [x] If this is the first time you're submitting a PR or even if you just need a refresher, consider reviewing our [style guide](https://github.com/dockstore/dockstore/wiki/Dockstore-Frontend-Opinionated-Style-Guide#pr-checklist)
- [x] Do not bypass Angular sanitization (bypassSecurityTrustHtml, etc.), or justify why you need to do so
- [x] If displaying markdown, use the `markdown-wrapper` component, which does extra sanitization
- [x] Do not use cookies, although this may change in the future
- [x] Run `npm audit` and ensure you are not introducing new vulnerabilities
- [x] Do due diligence on new 3rd party libraries, checking for CVEs
- [x] Don't allow user-uploaded images to be served from the Dockstore domain
- [x] If this PR is for a user-facing feature, create and link a documentation ticket for this feature (usually in the same milestone as the linked issue). Style points if you create a documentation PR directly and link that instead.
- [x] Check whether this PR disables tests. If it legitimately needs to disable a test, create a new ticket to re-enable it in a specific milestone.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -4,7 +4,7 @@
   "license": "Apache License 2.0",
   "config": {
     "webservice_version": "1.16.0-alpha.10",
-    "use_circle": true,
+    "use_circle": false,
```
Comment (coverbeck) [inline package.json:7]: Workaround for CircleCI artifact issue.

**Review by kathy-t — APPROVED** [review]

**Comment by kathy-t** [comment]
More general question, but going forward, should we avoid using flex layout for new pages in the UI?

**Comment by denis-yuen** [comment]
> More general question, but going forward, should we avoid using flex layout for new pages in the UI?

I'd guess so https://github.com/ngbracket/ngx-layout has a few linked alternatives. 

https://github.com/angular/flex-layout/issues/1426 has some discussion, @coverbeck how did you pick this particular one for the record?

**Comment by coverbeck** [comment]
> More general question, but going forward, should we avoid using flex layout for new pages in the UI?

I think we should keep using it for now. We don't know what we would do as an alternative, so it's probably best to replace it all at once.

**Comment by coverbeck** [comment]
> > More general question, but going forward, should we avoid using flex layout for new pages in the UI?
> 
> I'd guess so https://github.com/ngbracket/ngx-layout has a few linked alternatives.
> 
> [angular/flex-layout#1426](https://github.com/angular/flex-layout/issues/1426) has some discussion, @coverbeck how did you pick this particular one for the record?

* Option 1 was dropping flex-layout entirely, and replacing it with plain CSS. There were several posts, including variants using the tailwind.css library, on how you could do that.
* Option 2 was using a different library that was compatible with the existing. I saw 2 in the thread, the one I'm using here, and [this one](https://github.com/alessiobianchini/ng-flex-layout).

My CSS skills have faded, and the ones I had were flex layouts, so I didn't feel comfortable resolving option 1 in a short period of time. I feel it's more important to unblock the Angular upgrade; option 2 is the past of least resistance. In the future we might consider taking out flex-layout, but this is the less disruptive for now.

On why this library vs the other one, this one has a lot more activity.

**Review by svonworl — APPROVED** [review]

**Review by denis-yuen — APPROVED** [review]

**Pull request merged**